spec/03-plugins/37-opentelemetry/07-utils_spec.lua

```diff
@@ -0,0 +1,56 @@
+local helpers = require "spec.helpers"
+local LOG_PHASE = require("kong.pdk.private.phases").phases.log
+
+describe("compile_resource_attributes()", function()
+    local mock_request
+    local old_ngx
+    local old_kong
+    local compile_resource_attributes
+
+    setup(function()
+        old_ngx = _G.ngx
+        old_kong = _G.kong
```

Comment:
looks like we're not making changes to `_G.kong`, so we can remove this (and from lazy_teardown too)